app/Http/Controllers/Twill/BaseController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers\Twill;
+
+use A17\Twill\Models\Contracts\TwillModelContract;
+
+class BaseController extends ModuleController
+{
+    protected function setUpController(): void
+    {
+        $this->disablePermalink();
+        $this->enableSkipCreateModal();
```

Comment:
The permalinks don't always seem to work, and I truly dislike the create modal, so I've disabled them by default. We can re-enable these on a per-controller basis, as needed.